src/main/node/config-manager.ts

```diff
@@ -0,0 +1,152 @@
+var fs = require('fs');
+import isDirectory from './file-manager';
+import isPublication from './file-manager';
+
+class Colaborator {
+    username: string;
+    role: string;
+    constructor(username: string, role: string) {
+        this.username = username;
+        this.role = role;
+  }
+}
+
+export function createConfigFile(path: string, name: string, description: string, collaborators: Colaborator[], 
+                          packageManager: string, tag: string): boolean {
+  if (isDirectory(path)) {
+    let configContent = { 
+      "name": name,
+      "description": description,
+      "collaborants": collaborators,
+      "package_manager": packageManager,
+      "tag": tag 
+    };
+    let configContentJSON = JSON.stringify(configContent);
+    fs.writeFileSync(path + '/vn-pub.conf', configContentJSON);
+    return true;
+  }
+  return false;
+}
+
+export function deleteConfigFile(path: string): boolean {
+  if (isDirectory(path) && isPublication(path)) {
+    try {
+      if (fs.existsSync(path)) {
+        fs.unlinkSync(path + '/vn-pub.conf');
+        return true;
+      }
+    } catch(err) {
+      console.error(err);
+    }
+  return false;
+}
+
+export function modifyName(path: string, newName: string): boolean {
```

Comment:
Don't you think this function and other `modify` functions have a lot of code in common? If the filename changes then we have to remember to change it in every function. Same applies to other forms of handling the JSON